### PR TITLE
when not DEBUG set USE_DEBUG_HELPERS OFF

### PR DESCRIPTION
**Pull request by benbiti**
when not DEBUG set USE_DEBUG_HELPERS OFF

Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Review by kitaisreal — APPROVED** [review]

**Comment by kitaisreal** [comment]
@Mergifyio update

**Comment by kitaisreal** [comment]
@Mergifyio update

**Comment by kitaisreal** [comment]
@benbiti it seems like we need to fix build for fast test.

**Comment by benbiti** [comment]
@kitaisreal ok，i will fix it.

File: CMakeLists.txt [hunk 1]
```diff
@@ -257,7 +257,7 @@ endif()
 if (CMAKE_BUILD_TYPE_UC STREQUAL "DEBUG")
     set(USE_DEBUG_HELPERS ON)
 else ()
-    set(USE_DEBUG_HELPERS ON)
+    set(USE_DEBUG_HELPERS OFF)
```
Comment (alexey-milovidov) [inline CMakeLists.txt:260]: Why setting it off instead of doing nothing?

**Pull request merged**